src/main/java/com/google/growpod/data/Garden.java

```diff
@@ -33,12 +33,18 @@
   /** The garden's name. */
   private String name;
 
+  /** The garden's biography. */
+  private String biography;
```

Comment:
nit: `description` instead of `biography`?

Reply:
Will change.